docs/orchestration/concepts/api_keys.md

```diff
@@ -69,21 +75,22 @@ mutation {
 
 ## Querying for Key Information
 
-To query for information about a specific API key with GraphQL, execute the following query against `https://api.prefect.io`. This will allow you to query for API key information, however it never returns the value of the key itself.
+To query for information about API Keys with GraphQL, execute the following query against `https://api.prefect.io`. This will return API Key information, but never the value of the key.  All User API Keys for the active User will be returned, along with any Service Account API Keys which the API Key is privileged to see in the the API Key's default tenant.  Only and all Administrators are privileged to see all Service Accounts--because all Service Accounts have the Administrator role, they can also see all API Keys for any Service Account in the tenant.
```

Comment:
```suggestion
To query for information about API Keys with GraphQL, execute the following query against `https://api.prefect.io`. This will return API Key metadata, but the value of the key cannot be retrieved.
```
This next portion makes sense to me but is _very_ confusing. I'm not sure how to rewrite it.
`All User API Keys for the active user will be returned, along with any Service Account API Keys which the API Key is privileged to see in the the API Key's default tenant.  Only and all Administrators are privileged to see all Service Accounts--because all Service Accounts have the Administrator role, they can also see all API Keys for any Service Account in the tenant.`

Reply:
Completely agree--but the behaviour is confusing if we don't note SOMETHING here imho.  Leaving this suggestion open